sky/utils/cli_utils/cost_utils.py

```diff
@@ -0,0 +1,123 @@
+"""Utilities for sky cost report and sky spot cost."""
+from typing import Any, Dict, List
+import colorama
+from sky import global_user_state
+
+
+def get_cost_report(cluster_status):
+    cost = cluster_status['total_cost']
+
+    if not cost:
+        return '-'
+
+    return f'${cost:.3f}'
```

Comment:
Should we rename this function to `get_cost_from_record` and rename the argument to `record`?